src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -791,15 +791,41 @@ def _validate_lock_params_match_lock(
                 name, _resource_name))
 
 
-def get_lock(name, resource_group_name=None):
+def get_lock(name, resource_group_name=None, resource_provider_namespace=None,
+             parent_resource_path=None, resource_type=None, resource_name=None):
     """
-    :param name: Name of the lock.
+    :param name: The name of the lock.
     :type name: str
+    :param resource_provider_namespace: Name of a resource provider.
+    :type resource_provider_namespace: str
+    :param parent_resource_path: Path to a parent resource
```

Comment:
Help text should either be specified in `_help.py` (probably best) or the `params.py` file. This is the only way of adding help that we actively discourage (it is really meant for SDK reflection). 😇